Pixel Game/Assets/Scripts/DialogueViewer.cs

```diff
@@ -139,12 +139,14 @@ public void SwitchDialogueCanvasOnOff()
 
     void OnDisable()
     {
+        Time.timeScale = 1;

```

Comment:
What is this timescale in the dialogue viewer?

Reply:
It is set to zero when the dialog opens to pause the game.
I should have written a comment about that.

Reply:
So we can always set `Time.timeScale = 0;` to pause the game?

Reply:
Yep

Reply:
Nice, that is good to know